src/handlers.cpp

```diff
@@ -2543,11 +2545,52 @@ void PushProfileTask::on_get_irs_failure(CassandraStore::Operation* op,
 
 void PushProfileTask::update_reg_data()
 {
+  TRC_DEBUG("PPR is valid");
+  HTTPCode ret_code = 0;
```

Comment:
You only use this ret_code in the if statement below, so can you define it within that statement instead? Also, can this be rc or return_code, and can you set it to HTTP_OK to start with?